packages/fx-core/src/plugins/resource/aad/utils/telemetry.ts

```diff
@@ -28,18 +28,25 @@ export class TelemetryUtils {
   }
 
   public static sendException(
```

Comment:
I would suggest change function name to "sendErrorEvent"

Reply:
Fixed. Thanks!